eng/Build.props

```diff
@@ -1,12 +1,6 @@
 <Project>
 
-  <!-- VMR bootstrap -->
-  <ItemGroup Condition="'$(InitializeVMR)' == 'true'">
-    <ProjectToBuild Include="$(RepoRoot)src/VirtualMonoRepo/Tasks/VirtualMonoRepo.Tasks.csproj" BuildInParallel="false" />
-    <ProjectToBuild Include="$(RepoRoot)src/VirtualMonoRepo/InitializeVMR.proj" BuildInParallel="false" />
-  </ItemGroup>
```

Comment:
Do we not need this tooling here any more?

Reply:
Nope. Talked with Premek a few months ago. This isn't needed anymore as we won't initialize the VMR anymore again.